### PR TITLE
summer meetup info added

### PR DESCRIPTION
**Pull request by ipedrazas**

### CODE REVIEW

**Comment by ipedrazas** [comment]
@milosgajdos83 review, please 😄

File: content/post/kubernetes-summer.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+After that we will move to the next talk given by the one and only <drum rolls> [Tim Hockin](https://twitter.com/thockin),  Principal SW Engineer, Kubernetes & Google Container Engine, Google. Time will give a talk about Kubernetes networking, a topic which a lot of people tend to struggle with. Who else could shed more light on this than the man directly involved in the project straight from the start? :-)
+
+This Meetup would never happen without our wonderful sponsors. [Skills Matter](https://skillsmatter.com/) have kindly provided us with their excellent venue.
+
+Our best friends from [Jetstack](https://www.jetstack.io/), Kubernetes specialists based in London, will feed us before we head to the main room for the talks! <applause placeholder>. If you're looking into running Kubernetes in production you should definitely talk to Jetstack. They are quickly becoming the go-to experts in this space  So, do check their site and get in touch with them!
+
+## Program: 
+
+6:30 - 7:00 Drinks, pizza and socializing
```
Comment (ipedrazas) [inline content/post/kubernetes-summer.md]: I'm happy to add the logo of Jetstack...

File: content/post/kubernetes-summer.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+7:30 - 7:40 Q&A and short break 
+
+8:00 - 9:00 Tim Hockin, Deep-dive on Kubernetes networking 
+
+Kubernetes networking can feel a bit overwhelming for some people. In this talk Tim will walk us from the basics to the more advanced topics in Kubernetes networking. Time will explain some of the design decision which drove the networking architecture and much more.
+
+after 9:00PM: PUB
+
+## SPEAKER BIOS:
```
Comment (milosgajdos) [inline content/post/kubernetes-summer.md:41]: would it be worth adding their avatars / pics to this?

Comment (ipedrazas) [reply content/post/kubernetes-summer.md:41]: we should really create a speaker archetype... I would not do it now, though.

**Pull request merged**